src/web/app/pages-instructor/instructor-home-page/instructor-home-page.component.html

```diff
@@ -39,14 +39,14 @@ <h1>Home</h1>
   <div *tmIsLoading="!hasCoursesLoaded || isCopyLoading">
     <div *ngIf="courseTabModels.length > 0">
       <div id="course-tab-{{ idx }}" class="card" *ngFor="let courseTabModel of courseTabModels; let idx = index">
-        <div class="card-header bg-primary text-white cursor-pointer"
+        <div class="card-header bg-primary text-white cursor-pointer overflow-hidden"
```

Comment:
Just checking, is this addition needed? Seems to work fine without it. (Not sure if hidden is the right behaviour anyway - auto might be better)

Reply:
I thought I was appending to the body so I made it on purpose, but overflow-auto should also work as mentioned.
tested it & working as required

Reply:
I think in generally, we would rather have the user see the overflow, since at least they can still use the functionality.

In this case, can just remove the `overflow-` entirely. The default behaviour (i.e. no change) is fine

Reply:
if you wish , users to view the Overflow then we can remove the overflow- completely without disturbing the original functionality and it will work fine 
So I will remove the overflow

Reply:
> So I will remove the overflow

Great, thanks!